test/box-tap/extended_error.test.lua

```diff
@@ -145,6 +145,24 @@ test:isnt(err2, nil, 'Stack is received via iproto fields')
 test:ok(check_error(err1, checks1), "First error in the stack in iproto fields")
 test:ok(check_error(err2, checks2), "Second error in the stack in iproto fields")
 
+-- Check that call_16 never necodes errors in the extended format.
+local err = box.error.new(box.error.UNKNOWN)
```

Comment:
What is wrong with returning MP_ERROR from CALL_16? For instance, you didn't ban MP_UUID and MP_DECIMAL from it despite they are also added later than CALL_16. An error is also an object, just with a special type. If a client says he supports extended errors, I don't see why should they be banned from any response.

Reply:
You're right. I enabled MP_ERROR for CALL_16.